convert_results.py

```diff
@@ -0,0 +1,182 @@
+"""This script converts JSON result files to Markdown summaries.
+
+The JSON format is meant to contain all details from tests. For human readable
+outputs, use this script to extract the most interest information from your JSON
+reports and write them into an eye-friendly markdown file.
+
+  Typical usage example:
+
+  python convert_results.py --source=results/ --target=result_summaries/
+"""
+
+import argparse
+import json
+import os
+from tabulate import tabulate
+
+
+def summary_table(tests, passed_test_count, total_test_count):
+  """Returns a Markdown table summing passed tests for all tags."""
+  tags = {}
+  for test in tests:
+    pass_score = 1 if test['result'] == 'pass' else 0
+    for tag in test.get('tags', []):
```

Comment:
run flake8 to check compliance with pep8

Reply:
`flake8` enforces indentation spaces to be multiples of 4, which doesn't work with Google style. I used pylint instead, and configured it to use 2 spaces like:

> python3 -m pylint --indent-string='  ' convert_results.py